articles/tokens/access_token.md

```diff
@@ -1,49 +1,115 @@
+
+The third part of the JWT is the signature which is used by the recipient of a JWT to validate the integrity of the information conveyed in the JWT.
 
 The [Auth0.js documentation](/libraries/auth0js) shows a sample of how to get the `access_token`.
 
+::: warning-banner
+The `access_token` used to be an opaque string that was used to call the Auth0 API, and in some circumstances it still is. For more details on when the `access_token` is a JWT, and when it is an opaque string, please see [Access Token vs ID Token](/tokens/access_token-vs-id_token)
+
+!! Must still do this doc !!
+:::
+
+
 ## How to get Auth0 access token
 
 The Auth0 access token can be obtained in several ways.
 Calls to Lock or library functions that invoke authentication will return the `access_token`.
 
-* Calls to the Lock widget will return `access_token` as shown in the [Lock documentation](/libraries/lock)
-* [Examples using auth0.js](https://github.com/auth0/auth0.js)
-* The [/authorize endpoint in the authentication API](/auth-api) will return `access_token`
+* Calls to the Lock widget will return `access_token` as shown in the [Lock documentation](/libraries/lock) ?? Is this true - can we still use lock ??
+* [Examples using auth0.js](https://github.com/auth0/auth0.js) ?? is this still true ??
+* The [/i/oauth2/authorize endpoint in the authentication API](/auth-api) when using the Authorization Code Grant flow or Implicit Grant flow will return an `access_token`
+* The [?? endpoint in the authentication API](??) when using Client Credentials Grant will issue an `access_token`
 * Check the [List of tutorials](/tutorials) to see how to make calls to libraries for other languages/SDKs.
 
 ## How to control contents of access token
 
-It is not possible at this time to control the contents of the Auth0 access token.
+The contents of the `access_token`, specifically the claims contained within it, are controlled by the use of a parameter called `scope` which is passed to the authentication functions mentioned above.  For example, the call to the Lock widget’s `.show` function can specify optional authentication parameters as follows:
+
+?? Is this still true - can we use lock ??
+
+```
+lock.show({
+   responseType: ‘token’,
+   authParams: {
+      scope: ‘appointments contacts’
+    }
+ });
+```
+The above sample, specifying `appointments contacts` will result in a JWT with claims for accessing appointments and contacts. The responseType should be `token` for Implicit Grant flow and `code` for Authorization Code Grant flow.
+
+The scope of the `access_token` JWT can also be altered via Rules, through the context.jwtConfiguration.scopes object as documented [here](https://github.com/auth0/docs/blob/32033877180affa26233b8f65cb28bd532514eab/articles/rules/index.md#context) ?? Should be still suggest this. Also AFAIK rules won't work anymore with the new endpoint ??
```

Comment:
`context.jwtConfiguration.scopes` is most likely not going to effect the access_token, and will continue to add scope to the id_token.  This entire usage should be deprecated as it makes no sense.

It remains to be determined, but we will add rules support for modifying the access_token, but for backwards compatiblity this will be  a new property.  Probably something liek `context.accessToken.scopes`.